custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1247,6 +1247,24 @@
           "type": "property_name",
           "property_name": "child_name"
         }
+      },
+      {
+        "column_id": "location_rationalisation_date",
```

Comment:
is there a better phrase we can come up with to `location_rationalisation`? where did that come from? those words just don't fit together in any meaningful way in my brain and i imagine that will be likely to be true for anyone who visits this code later

Reply:
The phrase was chosen because it was the original name of this project and to be consistent with the case properties defined in https://docs.google.com/document/d/1h6kTr9M0FumswP3-gr_84OLeUHRmn5Y10dCpSAS3Y9g/edit#. It has since been renamed `location_reassignment`, and all of these will be updated to that (that change was quite recent. The change to the case property names were made in the last few days which is why it is not yet reflected